include/sys/euiAnimator.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <nn/font/font_Util.h>
+
+#include <nn/ui2d/ui2d_Animation.h>
+#include <nn/ui2d/ui2d_Group.h>
+#include <nn/ui2d/ui2d_Pane.h>
+
+#include <eui/lytex/euiLayoutEx.h>

```

Comment:
Don't include files that don't exist or create a dummy version of that header with a fwd decl, also the include isn't needed here just do a fwd decl of the class